### PR TITLE
fix: Attendee form questions order in order-form

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5853

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/orders/order-form.js [hunk 1]
```diff
@@ -497,7 +497,10 @@ export default Component.extend(FormMixin, {
   },
 
   allFields: computed('fields', function() {
-    return groupBy(this.fields.toArray(), field => field.get('form'));
+    const requiredFixed = this.fields.toArray().slice(0, 3);
+    const customFields =  orderBy(this.fields.toArray().slice(3), ['isComplex', 'name'], ['asc', 'asc']);
```
Comment (daretobedifferent18) [inline app/components/forms/orders/order-form.js:501]: by default all values are sorted in ascending order

Comment (maze-runnar) [reply app/components/forms/orders/order-form.js:501]: no they are not in a way in attendee form.

Comment (daretobedifferent18) [reply app/components/forms/orders/order-form.js:501]: I mean orderBy funtion by default I think sort all values in ascending order so no need to pass ```['asc', 'asc']```

File: app/components/forms/orders/order-form.js [hunk 2]
```diff
@@ -497,7 +497,10 @@ export default Component.extend(FormMixin, {
   },
 
   allFields: computed('fields', function() {
-    return groupBy(this.fields.toArray(), field => field.get('form'));
+    const requiredFixed = this.fields.toArray().slice(0, 3);
```
Comment (iamareebjamal) [inline app/components/forms/orders/order-form.js:500]: There is no guarantee that only first three fields are fixed fields, the logic is flawed

Comment (maze-runnar) [reply app/components/forms/orders/order-form.js:500]: this is for firstName, lastName and email. They are always at top?

Comment (maze-runnar) [reply app/components/forms/orders/order-form.js:500]: the rest in order in which they are in attendee step

Comment (iamareebjamal) [reply app/components/forms/orders/order-form.js]: But they'll not always be 3

**Comment by iamareebjamal** [comment]
Test is failing

**Comment by maze-runnar** [comment]
> Test is failing

I am not understanding why test are failing and why they are failing in that particular test. I mean they don't seem even related :confused:

**Comment by iamareebjamal** [comment]
It's issue with heroku server used to test integration tests. Fixing

**Pull request merged**